static/src/javascripts/projects/commercial/modules/sticky-mpu.spec.ts

```diff
@@ -44,16 +41,17 @@ describe('Sticky MPU', () => {
 
 	it('should resize the parent container', (done) => {
 		mockHeight(8000);
-		const targetSlot = document.querySelector('.js-ad-slot');
-		targetSlot.dataset = {
-			name: targetSlot.getAttribute('data-name') || '',
-		};
-		mediator.once('page:commercial:sticky-mpu', () => {
-			const container = document.querySelector('.aside-slot-container');
-			expect(container.style.height).toBe('8000px');
-			done();
-		});
-		stickyMpu(targetSlot);
+		const targetSlot = document.querySelector<HTMLElement>('.js-ad-slot');
+		if (targetSlot) {
```

Comment:
It might be worth asserting that `targetSlot` isn't null to avoid the possibility of the test continuing to pass without _really_ testing anything if the selector and classname `js-ad-slot` are ever misaligned in the future.
```suggestion
		expect(targetSlot).not.toBeNull();
		if (targetSlot) {
```

Reply:
Ah yeah, nice catch!